pkg/scheduler/framework/plugins/imagelocality/image_locality_test.go

```diff
@@ -327,6 +367,55 @@ func TestImageLocalityPriority(t *testing.T) {
 			expectedList: []framework.NodeScore{{Name: "node1", Score: 1}, {Name: "node2", Score: 0}},
 			name:         "pod with multiple small images",
 		},
+		{
+			// Pod: gcr.io/30 gcr.io/40
+
+			// Node1
+			// Image: gcr.io/20:latest 20MB, gcr.io/30:latest 30MB gcr.io/40:latest 40MB
+			// Score: 100 * (30M + 40M * 1/2 - 23M) / (1000M * 2 - 23M) = 1
+
+			// Node2
+			// Image: 100 * (30M - 23M) / (1000M * 2 - 23M) = 0
+			// Score: 0
+			pod:          &v1.Pod{Spec: test3040},
+			nodes:        []*v1.Node{makeImageNode("node1", node203040), makeImageNode("node2", node400030)},
+			expectedList: []framework.NodeScore{{Name: "node1", Score: 1}, {Name: "node2", Score: 0}},
+			name:         "pod with multiple small images",
+		},
+		{
+			// Pod: gcr.io/30 (Always) gcr.io/40 (Always)
+
+			// Node1
+			// Image: gcr.io/20:latest 20MB, gcr.io/30:latest 30MB gcr.io/40:latest 40MB
+			// Score: 0 + 0 = 0
+
+			// Node2
+			// Image: gcr.io/4000:latest 4000MB, gcr.io/30:latest 30MB
+			// Score: 0 + 0 = 0
+			pod:          &v1.Pod{Spec: test3040_always3040},
+			nodes:        []*v1.Node{makeImageNode("node1", node203040), makeImageNode("node2", node400030)},
+			expectedList: []framework.NodeScore{{Name: "node1", Score: 0}, {Name: "node2", Score: 0}},
+			name:         "pod with multiple small images and all ImagePullPolicy set to Always",
+		},
+		{
```

Comment:
Is this testcase's result the same before or after the PR, if so, I think we should add a better one. Like before we prefer node1, after the PR, then node2. I didn't test it myself, seems the same result.

Reply:
Deleted the duplicate testcase.